### PR TITLE
MGDCTRS-461: feat: Provide a dialog to opt-in to the eval namespace, …

### PR DESCRIPTION
**Pull request by indraraj**
Closes issue 
https://issues.redhat.com/browse/MGDCTRS-461

**Eval Register dialog**
image
**Eval in provisioning state** (provisioning namespace can not be selected to move ahead)
image
**Eval in deleting state**
image

### CODE REVIEW

**Comment by heiko-braun** [comment]
Otherwise it looks good!

**Comment by heiko-braun** [comment]
@gashcrumb Mind a review?

File: locales/it/public.json [hunk 1]
```diff
@@ -102,6 +106,8 @@
   "ownerSearchPlaceholder": "Search by owner",
   "Create Kafka instance": "Create Kafka instance",
   "namespaceStepDescription": "Connectors can be installed on your own OSD cluster(please find the Add on installation guide) or Red Hat own OSD for evaluation purposes.",
+  "evalNamespaceExpire": "The eval connectros instance and its namesace will expire in ",
```
Comment (gashcrumb) [inline locales/it/public.json:109]: typos here

File: locales/en/public.json [hunk 2]
```diff
@@ -102,6 +106,8 @@
   "ownerSearchPlaceholder": "Filter by owner",
   "Create Kafka instance": "Create Kafka instance",
   "namespaceStepDescription": "Connectors can be installed on your own OSD cluster(please find the Add on installation guide) or Red Hat own OSD for evaluation purposes.",
+  "evalNamespaceExpire": "The eval connectros instance and its namesace will expire in ",
```
Comment (gashcrumb) [inline locales/en/public.json:109]: typos here

File: locales/en/public.json [hunk 3]
```diff
@@ -62,6 +63,9 @@
   "namespaceExpireAlert": "The eval namespace has a duration of 48 hours. The Connector hosted on this namespace will expire when the namespace are not available.",
   "registrationEvalNamespace": "Registration of eval namespace",
   "register": "Register",
+  "namespaceInformation": "Namespace information",
+  "Duration": "Duration",
+  "namespaceModelAlert": "The namespace will be ready for shortly after registration.",
```
Comment (gashcrumb) [inline locales/en/public.json:68]: "ready shortly after registration"

File: src/app/pages/CreateConnectorPage/StepNamespace.tsx [hunk 4]
```diff
@@ -42,24 +66,43 @@ export function SelectNamespace() {
...
   } = useNamespaceMachine();
   const onModalToggle = useCallback(() => {
     setIsModalOpen((prev) => !prev);
   }, []);
 
+  const refreshResponse = () => {
+    onRefresh();
+  };
+
+  const onNamespaceSelection = (namesace: ConnectorNamespace) => {
```
Comment (gashcrumb) [inline src/app/pages/CreateConnectorPage/StepNamespace.tsx:94]: typo "namespace" and in this function

**Comment by gashcrumb** [comment]
Tested out fine here, other than the nits I think it's good to go.  Thanks!

**Review by gashcrumb — APPROVED** [review]

File: locales/en/public.json (hunk 2 above)
Comment (indraraj) [reply locales/en/public.json]: done

File: locales/en/public.json (hunk 3 above)
Comment (indraraj) [reply locales/en/public.json]: done

File: src/app/pages/CreateConnectorPage/StepNamespace.tsx (hunk 4 above)
Comment (indraraj) [reply src/app/pages/CreateConnectorPage/StepNamespace.tsx]: done

**Pull request merged**